src/content/docs/contributing/client-api/service-version.mdx

```diff
@@ -118,6 +118,111 @@ package:
 
             **Implementation note:** Wrapping the generated stubs is not ideal, but if it has to be done,
             ensure the wrapper is consistent with the stub API.
+
+  functions:
+    - name: get_root_workspace
+      description: |
+        Gets the root workspace for the specified organization.
+        This function queries RBAC v2 to find the root workspace for the given org_id.
+        
+        GET /v2/workspaces?type=root
+        
+        Example:
+        
+        ```python
+        import requests
+        
+        # Configure HTTP client with authentication headers
+        http_client = requests.Session()
+        http_client.headers.update({
+            "x-rh-rbac-psk": "your-pre-shared-key",
+            "x-rh-rbac-client-id": "client-id"
+        })
```

Comment:
This should no longer be required, as RBAC now accepts JWT auth, meaning teams can use the same token flow as they do for other Kessel client calls. We may want to update this documentation with an example as well:

- https://github.com/project-kessel/docs/blob/94665e64476db6ce5b2b365208d0eff7f76f57eb/src/content/docs/building-with-kessel/how-to/migrate-from-rbac-v1-to-v2.mdx#enabling-authentication

Reply:
Ah perfect, I wasn't sure if that was the case. We should be able to utilize the existing oauth credentials helpers here then